### DISCUSSION TITLE
Will my school mind if I upgrade the school chromebook?

### DISCUSSION THREAD

**Discussion by RavenChill-Dev** [3kh0/ext-remover#658] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Seeing how I built my own PC before, I want to upgrade this metal box called a "chromebook" that is completely useless into a powerhouse windows 11 laptop. If I do, I will make it 64 gb RAM (memory), Amazing graphics, new LED RGB Keyboard, and a better camera, BTW this is the chromebook 11a

**Comment by RavenChill-Dev** [comment]
Lol

**Comment by Hilololsusy** [comment, marked as answer]
why are you asking us go ask your adminstator also why invest so much money into something you don't keep

**Reply by RavenChill-Dev** [reply]
Oh im not paying for it, the school is.
